src/components/Media/InfoPanel.js

```diff
@@ -0,0 +1,62 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+import { MdCheck, MdClose, MdDelete, MdPlayArrow, MdStar } from "react-icons/md";
+
+const useStyles = makeStyles((theme) => ({
+  line: {
+    display: "flex",
+  },
+  icon: {
+    fontSize: "1.5rem",
+  },
+  text: {
+    width: "95%",
```

Comment:
Du weißt, dass wir die Wrapper Komponente besitzen? Ich denke dann brauchst du das nicht 😃

Reply:
Ja, hab ich getestet - weil Komponente in Komponente, macht der Wrapper nicht was er soll.